### PR TITLE
First screen

### PR DESCRIPTION
**Pull request by radishevskyi**
# 📝 Description

- 

# 🔄 Type of change

- [ ] Cleaning up
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality to something)
- [ ] Configuration change
- [ ] Translation change

# 🧪 How has this been tested / or will it be tested?

- [ ] Manually surfing the app

# ✅ Checklist:

- [ ] My changes generate no new warnings in console or terminal
- [ ] I have performed a self-review of my code
- [ ] I have updated versions of the package.json and package-lock.json

# 🖼️ Result screenshot/video (optional):

# Gif to summarize my emotions (optional):

### CODE REVIEW

File: src/modules/home/first-screen/first-screen.styles.ts [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+  `,
+
+  Button: styled.button`
+    display: flex;
+    gap: 10px;
+    background: #415346;
+    padding: 20px 40px;
+    border-radius: 100px;
+    cursor: pointer;
+    font-size: 18px;
```
Comment (p-osypov) [inline src/modules/home/first-screen/first-screen.styles.ts:60]: Add css variable --font-size-medium: 1.125rem; /* 18px */
```suggestion
font-size: var(--font-size-medium);
```

File: src/modules/home/first-screen/first-screen.tsx [hunk 2]
```diff
@@ -0,0 +1,31 @@
+import Icons from '@/shared/assets/icons';
+import { SC } from './first-screen.styles';
+
```
Comment (p-osypov) [inline src/modules/home/first-screen/first-screen.tsx:3]: Create translations for this section

File: src/modules/home/first-screen/first-screen.tsx [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+        <SC.UnitName>103 ОБР ТРО Чорний стриж</SC.UnitName>
+        <SC.Title>
+          Кожен хід армії <br /> Сила в кожній місії
+        </SC.Title>
+        <SC.Subtitle>
+          Після повномасштабного вторгнення ми мобілізували свої зусилля.
+        </SC.Subtitle>
+        <SC.Button>Приєднатися до нас</SC.Button>
+        <SC.Contacts>
+          <SC.ContactLink href="tel:(209) 555-0104">
```
Comment (p-osypov) [inline src/modules/home/first-screen/first-screen.tsx:17]: Create /src/shared/config.ts with const contacts where we gonna keep all meta information of this project.

File: src/modules/home/first-screen/first-screen.tsx [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+            <Icons.PhoneNum style={{ marginRight: '10px' }} />
```
Comment (p-osypov) [inline src/modules/home/first-screen/first-screen.tsx:18]: Move inline styles to styled-components, or add class icon to these icons and set margin in ContactLink .icon.


-------------
If design has non `--spacing` gap we should ignore it and use `--spacing` that is near to value from design. 
In that case marginRight must be `var(--spacing)`.

File: src/shared/assets/icons/mail.tsx [hunk 5]
```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { IconProps } from '@/shared/assets/icons/icons.types';
+
+const Email = ({ color, ...props }: IconProps) => (
+  <svg
+    width="16"
```
Comment (p-osypov) [inline src/shared/assets/icons/mail.tsx:6]: set width and height 20px to all icons by default

File: src/shared/assets/icons/mail.tsx [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+    <defs>
+      <linearGradient
+        id="paint0_linear_1_14630"
+        x1="-6.5"
+        y1="0.500002"
+        x2="17.8771"
+        y2="1.64803"
+        gradientUnits="userSpaceOnUse"
+      >
+        <stop stop-color="#FAFAFA" />
```
Comment (p-osypov) [inline src/shared/assets/icons/mail.tsx:26]: Colors must be set from props.
```suggest
<stop stopColor={color || "#FAFAFA"} />
```

--------------


Update all icons we have. Check the console, there could be errors related to invalid svg properties.

**Review by p-osypov — CHANGES_REQUESTED** [review]

File: src/modules/home/first-screen/first-screen.styles.ts [hunk 7]
```diff
@@ -0,0 +1,81 @@
...
+  `,
+
+  Content: styled(Container)`
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    justify-content: center;
+    padding-bottom: var(--spacing-x4);
+    min-height: 100vh;
+    padding-top: calc(106px + var(--spacing-x2));
```
[the remark this answers is not in the document]
Comment (radishevskyi) [reply src/modules/home/first-screen/first-screen.styles.ts]: [fix: styles](https://github.com/p-osypov/army/pull/3/commits/6853d1e7640ca1fba6c6ca3378fea9345322bc58)

File: src/modules/home/first-screen/first-screen.styles.ts [hunk 8]
```diff
@@ -0,0 +1,81 @@
...
+  `,
+
+  UnitName: styled.span`
+    display: block;
+    border: 1px solid #41534640;
+    background: #41534633;
```
[the remark this answers is not in the document]
Comment (radishevskyi) [reply src/modules/home/first-screen/first-screen.styles.ts]: [fix: styles](https://github.com/p-osypov/army/pull/3/commits/6853d1e7640ca1fba6c6ca3378fea9345322bc58)

File: src/modules/home/first-screen/first-screen.styles.ts [hunk 9]
```diff
@@ -0,0 +1,81 @@
...
+  `,
+
+  Subtitle: styled.p`
+    margin-bottom: calc(var(--spacing-x3) * 2);
+    font-size: 20px;
+  `,
+
+  Button: styled.button`
+    display: flex;
+    gap: 10px;
```
[the remark this answers is not in the document]
Comment (radishevskyi) [reply src/modules/home/first-screen/first-screen.styles.ts]: [fix: styles](https://github.com/p-osypov/army/pull/3/commits/6853d1e7640ca1fba6c6ca3378fea9345322bc58)

File: src/modules/home/first-screen/first-screen.styles.ts (hunk 1 above)
Comment (radishevskyi) [reply src/modules/home/first-screen/first-screen.styles.ts]: [fix: styles](https://github.com/p-osypov/army/pull/3/commits/6853d1e7640ca1fba6c6ca3378fea9345322bc58)

File: src/modules/home/first-screen/first-screen.tsx (hunk 2 above)
Comment (radishevskyi) [reply src/modules/home/first-screen/first-screen.tsx:5]: [feat: translation for first-screen](https://github.com/p-osypov/army/pull/3/commits/0a648767a748014f5d1b1d5ad32a3db8ccf464a4)

File: src/modules/home/first-screen/first-screen.tsx (hunk 3 above)
Comment (radishevskyi) [reply src/modules/home/first-screen/first-screen.tsx]: [feat: created obj contacts](https://github.com/p-osypov/army/pull/3/commits/0851545ecc98bd641064dd0f3456c0895de4853f)

File: src/modules/home/first-screen/first-screen.tsx (hunk 4 above)
Comment (radishevskyi) [reply src/modules/home/first-screen/first-screen.tsx]: [fix: logic](https://github.com/p-osypov/army/pull/3/commits/9ded7df66ab5bb9237f6e0d9310e4d7c1a9cdc48)

**Review by p-osypov — APPROVED** [review]

**Pull request merged**